### PR TITLE
Add status colors for Maintenance, Provision, and Lease duration states in UI

### PR DESCRIPTION
**Pull request by skanthed**

### CODE REVIEW

File: esi_ui/static/dashboard/esi/lessee/nodes/esi-nodes-table.html [hunk 1]
```diff
@@ -114,18 +115,24 @@
               {$ node.power_state + (node.target_power_state ? " → " + node.target_power_state : "") $}
             </div>
           </td>
-          <td class="rsp-p2">{$ node.maintenance $}</td>
+          <td class="rsp-p2" ng-style="{'color': node.maintenance === 'True' ? 'red' : 'green'}">
```
Comment (tzumainn) [inline esi_ui/static/dashboard/esi/lessee/nodes/esi-nodes-table.html]: I'd suggest creating a `getMaintenanceColor` function and using that here; I know that the code is simple, but it creates a consistency (and makes it easier to update the colors in one place if needed later).

Comment (skanthed) [reply esi_ui/static/dashboard/esi/lessee/nodes/esi-nodes-table.html]: It was very simple so I did this way for other two I did with functions, will do it that for maintenance too.

File: esi_ui/static/dashboard/esi/lessee/nodes/esi-nodes-table.controller.js [hunk 2]
```diff
@@ -364,6 +368,35 @@
...
+
+    function getProvisionStateColor(provisionState, targetProvisionState) {
+      if (targetProvisionState) {
+        return 'blue';
+      }
+      if (PROVISION_ERROR_STATES.has(provisionState)) {
+        return 'red'; 
+      }
+      if (PROVISION_STABLE_STATES.has(provisionState)) {
+        return provisionState === 'active' ? 'green' : 'brown';
```
Comment (tzumainn) [inline esi_ui/static/dashboard/esi/lessee/nodes/esi-nodes-table.controller.js]: Can we change this brown to also blue? The brown looks pretty alarming, making it look like there might be something wrong with the node.

**Review by tzumainn — CHANGES_REQUESTED** [review]
One minor comment; looks good otherwise!

File: esi_ui/static/dashboard/esi/lessee/nodes/esi-nodes-table.controller.js (hunk 2 above)
Comment (skanthed) [reply esi_ui/static/dashboard/esi/lessee/nodes/esi-nodes-table.controller.js]: Even I felt that way. I tried differnt color but didn't suit. So I kept it brown in the end. Changed to blue now.

**Review by tzumainn — APPROVED** [review]
Looks good - thanks!

**Pull request merged**